website/emp-laboratories.md

```diff
@@ -0,0 +1,17 @@
+Jack A Gilbert
+--------------
+
+Argonne National Laboratory, 9700 S Cass Avenue, Lemont, IL, 60439, USA
+gilbertjack@anl.gov
+
+Rob Knight
+----------
+
+Department of Chemistry and Biochemistry, University of Colorado at
```

Comment:
Hi Colin, I made that change in a pull request *on your branch*. Or at least that's what I tried to do. Does that make sense?